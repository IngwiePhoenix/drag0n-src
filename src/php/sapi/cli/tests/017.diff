008- 
018- 
029- 
037- 