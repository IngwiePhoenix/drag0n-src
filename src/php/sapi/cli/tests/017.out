--------------
Snippet no. 1:
--------------
Interactive shell

Hello world

--------------
Snippet no. 2:
--------------
Interactive shell

multine
single
quote

--------------
Snippet no. 3:
--------------
Interactive shell

Here
comes
the
doc

--------------
Snippet no. 4:
--------------
Interactive shell

Done

--------------
Snippet no. 5:
--------------
Interactive shell


Parse error: syntax error, unexpected ')' in php shell code on line 1


Done